### PR TITLE
Added-PR-template

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
This PR will resolve issue #2 .

### CODE REVIEW

File: .github/PR_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#### Issue Number
```
Comment (khushishikhu) [inline .github/PR_template.md:1]:
```suggestion
#### Description
```
remove this line

File: .github/PR_template.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+fixes #
```
Comment (khushishikhu) [inline .github/PR_template.md:2]:
```suggestion
Issue FIXES #
```

**Review by khushishikhu — CHANGES_REQUESTED** [review]

**Comment by khushishikhu** [comment]
@96RadhikaJadhav I have suggested few changes rest is perfect. And right now I can't merge it as the coding period isn't started yet. First you will get assigned to that issue and the this PR will be considered for evaluation.

**Comment by khushishikhu** [comment]
@96RadhikaJadhav I have requested some changes here. Please do that changes and rest is all perfect.

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav I have requested some changes here. Please do that changes and rest is all perfect.

@khushishikhu  Done.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#### Description
+Iusse FIXES #
```
Comment (khushishikhu) [inline .github/pull_request_template.md:2]:
```suggestion
Issue FIXES #
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+<!-- Please Mention the issue number as  ISSUE #(Issue Number)
+Example:
+fixes #5
+-->
+
+#### Describe the changes you've made
+A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!
```
Comment (khushishikhu) [inline .github/pull_request_template.md:9]: @96RadhikaJadhav please comment out the statement of every description

**Review by khushishikhu — CHANGES_REQUESTED** [review]

**Review by khushishikhu — APPROVED** [review]

**Pull request merged**